### DISCUSSION TITLE
Netgear A6100 Working

### DISCUSSION THREAD

**Discussion by serbimac** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#253] [Show and tell]
Netgear A6100 confirmed working on both 2.4GHz and 5GHz networks for macOS 11.5.2.  Thanks for the hard work I was looking so long for a solution to get this adapter working!

**Comment by chris1111** [comment]
Thanks for report, its a nice adapter, very good.
[Working List Netgear A6100](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/commit/623a7626eda1c431193f15ba8190ead856cd35a5)
